test/test_routes.py

```diff
@@ -54,6 +54,15 @@ def test_get_config_route(self, test_client):
         response = test_client.get("/config/")
         assert response.status_code == 200
 
+    def test_upload_route(self, test_client):
+        response = test_client.post(
+            "/upload", files={"files": open("test/test_files/alphabet.txt", "r")}
+        )
+        assert response.status_code == 200
+        files = response.json()
+        with open(files[0]) as saved_file:
+            assert saved_file.read() == "abcdefghijklmnopqrstuvwxyz"
```

Comment:
You may want to add a test to ensure that the temporary file name is what you expect

I also I would add a test containing a larger file. I guess it doesn't have to be massive but maybe more than 1024 bytes so that we know that the chunking is accurate.